runtime/doc/help.txt

```diff
@@ -164,13 +167,24 @@ GUI ~
 
 Interfaces ~
 |if_cscop.txt|	using Cscope with Vim
+|if_perl|	Perl interface
```

Comment:
The others in this list contain the `.txt` extension, should this one too?

Reply:
Yep. Thanks.